client/components/event-details/SimilarEvents.tsx

```diff
@@ -0,0 +1,54 @@
+import styled from 'styled-components'
```

Comment:
There's a future discussion to be had about whether we should do this in the FE or BE, and what exactly makes two events "similar". Nice filler implentation though!